scripts/src/main/resources/scripts/command/node

```diff
@@ -48,24 +69,37 @@ then
 fi
 
 #CLI
-if [ "${1}" = "-h" ] || [ "${1}" = "help" ]
-then
-  echo "Setup or run yarn."
+case ${1} in 
+"help" | "-h")
```

Comment:
nice. I should think about using this syntax in all commandlets as it is more compact. Thanks! 👍